sdk/v1/client_test.go

```diff
@@ -316,14 +321,19 @@ func TestClient_PostTasksCount(t *testing.T) {
 		searchReindexClient := newSearchReindexClient(t, httpClient)
 
 		Convey("When search-reindexClient.PostTasksCount is called", func() {
-			respETag, task, err := searchReindexClient.PostTasksCount(ctx, headers, testJobID, testPayload)
+			respHeaders, task, err := searchReindexClient.PostTasksCount(ctx, headers, testJobID, testPayload)
```

Comment:
This is not really in this PR but please would you mind changing line 319 to be 404 instead of 409:

			Convey("Given a 404 response", t, func() {